ofnetAgent.go

```diff
@@ -127,16 +128,17 @@ const (
 // Create a new Ofnet agent and initialize it
 /*  routerInfo[0] -> Uplink nexthop interface
  */
-func NewOfnetAgent(bridgeName string, dpName string, localIp net.IP, rpcPort uint16,
```

Comment:
Based on discussion with @abhinandanpb the restart of network driver is currently requiring all the configs on the network to be deleted during the fwd mode change. This is probably not required for arp mode change. Will make that change